config.env

```diff
@@ -0,0 +1,9 @@
+NODE_ENV = development
+PORT = 5020
+USER = ncutixavier
+PASSWORD = 12345678
+DATABASE_URI=mongodb+srv://ncutixavier:<PASSWORD>@cluster0.o8egx.mongodb.net/portifolio?retryWrites=true&w=majority
+DATABASE = mongodb://localhost:27017/portifolio
+DATABASE_PASSWORD = DebITMmwuSTlGiId
+JWT_SECRET=this-is-secured-secret-and-insecured-secret
+JWT_EXPIRES_IN=90d
```

Comment:
Environmental variables are not pushed to github because it means anyone who has access to the repository can view your credentials. Change the values to sample values for example, `USER= //your username`